docs/DeveloperGuide.md

```diff
@@ -639,7 +674,45 @@ if there is no existing same item. If there is an existing same item,
    1. Other incorrect delete commands to try: `delete-i`, `delete-i x`, `...` (where x is larger than the list size or 
    x is a negative number)<br>
       Expected: Similar to previous.
+      
+### Undoing a command
 
+1. Undoing a command before any commands have been entered.
+    
+    1. Prerequisites: No commands have been entered yet.
+    
+    1. Test case: `undo` <br>
+        Expected: Error message is shown, stating that undo cannot be performed.
+        
+1. Undoing after a command has executed
+
+    1. Prerequisites: The last command entered was `clear-i`, which cleared all of 5 items in the inventory book.
+    
+    1. Test case: `undo` <br>
+        Expected: The inventory book is restored to the state where it had 5 items. Success message is displayed.
+      
+### Redoing an undone command
+
+1. Redoing a command before any commands have been undone.
+    
+    1. Prerequisites: No commands have been undone yet.
```

Comment:
Since this section is meant for testers to test manually, and since for this mod the tester is new for our app (PE), would you think putting a **Note: you can restart the application if you have entered commands previously** would be helpful for the tester?

Reply:
can